framework/src/components/storage/entities/transaction.js

```diff
@@ -243,7 +243,7 @@ class Transaction extends BaseEntity {
 			fieldName: 't_amount',
 		});
 		this.addField('fee', 'string', {
-			filter: filterTypes.NUMBER,
+			filter: filterTypes.TEXT,
```

Comment:
It should be consistent for every field. e.g. `reward` and `fee` fields both have filter type `NUMBER`, why we are changing this for one and not for other? 

This could cause different filter list on the entity level. e.g. With this change there will be no `fee_gt` and similar filters. The application may be using, so we should avoid that change. 

If its required we should come up with some better explanation for it.